docker/jupyterhub/jupyterhub_config.py

```diff
@@ -95,7 +101,7 @@
     "image":
     sqlflow_mysql_image,
     "imagePullPolicy":
-    "Always",
```

Comment:
Maybe we can keep the imagePullPolicy to be  `Always` so that we can use the latest SQLFlow image.

Reply:
I will setup a polling task to do this, `Always` makes user wait a long time for start the playground.